other-docs/guides/migrating-from-wordpress.md

```diff
@@ -0,0 +1,150 @@
+# Migrating a WordPress Codebase
+
+This guide covers how to migrate a typical WordPress project to HM Platform.
+
+### Setup Composer
+
+If the project does not already have a `composer.json` in the root of the repository, you should run `composer init` from the project root. This will prompt for any fields. For now, don’t bother adding any dependancies. If you already have a `composer.json` move to Remove Dependencies.
```

Comment:
```suggestion
If the project does not already have a `composer.json` in the root of the repository, you should run `composer init` from the project root. This will prompt you fill out the required fields. For now, don’t bother adding any dependencies. If you already have a `composer.json` move to Remove Dependencies.
```

Reply:
Guess we don't have header anchors to link to yet. That'd be a good issue to add

Reply:
Handled this as part of #48